sites/partners/src/components/listings/PaperListingForm/UnitGroupForm.tsx

```diff
@@ -573,7 +573,13 @@ const UnitGroupForm = ({
           {t("t.saveExit")}
         </Button>
 
-        <Button type="button" onClick={() => onClose()} variant="text" size="sm">
+        <Button
+          type="button"
+          onClick={() => onClose()}
+          variant="text"
+          size="sm"
+          className={"font-semibold darker-alert"}
```

Comment:
I'm assuming this is intentional, but I noticed this button didn't have the "semi-bold" class before this change

Reply:
Yep yep! This was just inconsistent